docs/release/trg-0/trg-7-4.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TRG 7.04 - IP checks for 3rd party content
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: "mandatory after": 19th of May 2023
+:::
+
+| Status | Created     | Post-History  |
+|--------|-------------|---------------|
+| Draft  | 13-Apr-2023 | Moved from OSS Development           |
+
+## Why
+
+Eclipse Tractus-X is an open source project hosted by the Eclipse Foundation licensed under the [Apache License 2.0](https://spdx.org/licenses/Apache-2.0). The legal obligations of the content must be observed in all forms of which the content is available.
+
+This page contains information about legal obligations and checks of your 3rd party content. The reference is always the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#ip-third-party).
+
+:::info
+
+The requirements described here must be met for each contribution.
+
+:::
+
+## Description
+
+The term third-party content refers to any content that is leveraged by the Eclipse project that is not produced and/or maintained as an asset of the project. This includes content from other Eclipse projects. See the complete explanation of [third-party content](https://www.eclipse.org/projects/handbook/#ip-third-party).
+
+All third-party content has to be checked and approved by the Eclipse Foundation IP Team. There are two ways:
+
+- Creating an IP issue manually (e.g. fonts, images, ...);
+- Using the Eclipse Dash License Tool to creat IP issues in an automated way (libraries).
+
+All third party content used has to be documented in the NOTICE file or the DEPENDENCY file. (TODO AWI: add link to TRG 7.01)
```

Comment:
"in the notice file or in the dependency file" not sure but i think a second 'in' might be good before the dependency file.